apache-rat-core/src/main/java/org/apache/rat/ReportConfiguration.java

```diff
@@ -121,6 +127,22 @@ public void setArchiveProcessing(Processing archiveProcessing) {
         this.archiveProcessing = archiveProcessing;
     }
 
+    /**
+     * Retrieves the archive processing type.
+     * @return The archive processing type.
+     */
+    public Processing getStandardProcessing() {
+        return standardProcessing == null ? Defaults.STANDARD_PROCESSING : standardProcessing;
+    }
+
+    /**
+     * Sets the archive processing type.  If not set will default to NOTIFICATION.
```

Comment:
I do not see this default setting in the code or is it handled somewhere else as "if null NOTIFICATION"?

Reply:
Must have been to late when I did the review. Thanks.